_src/_assets/js/d3-area-chart.js

```diff
@@ -133,5 +134,21 @@ function d3AreaChart({
     .attr('opacity', 0.9)
     .attr('fill', fillArgument)
 
+  if (annotations) {
```

Comment:
this will always be true unless someone explicitly passes `annotations:null` to the function.
perhaps better to check use `if(annotations.length)` or pass `null` as default value for `annotations`.

Reply:
you could also use `if(annotations && annotations.length)` to prevent this block from being run with an empty array.  Don't think it matters that much because it does nothing if passed an empty array, just for code cleanliness.